docs/user/document/tracking/initial_release_date-usage.en.md

```diff
@@ -4,7 +4,8 @@ The date when the first version of the document was published.
 In pre-release versions the planned publication date is used.
 
 The value should not be older than the `date` of the oldest item in Revision History (`/document/tracking/revision_history`).
-Typically, this is identical to the oldest element in the revision history, version 1.
-The value of initial release date does not change after publication of that first version anymore.
+Typically, this is identical to the oldest element in the revision history.
+That is, `1` in case of integer versioning respectively `1.0.0` in case of semantic versioning.
```

Comment:
It still took me a moment to understand what you wanted to say. Let me try something:
```suggestion
For published documents, the oldest item has the `number` value of `1` in case of integer versioning respectively `1.0.0` in case of semantic versioning.
```
If you come up with better language - feel free

Reply:
I tried to simplify a little more